src/transformers/modeling_utils.py

```diff
@@ -2704,17 +2707,6 @@ def from_pretrained(
                 )
 
         quantization_method_from_args = None
-        if quantization_config is not None:
-            quantization_method_from_args = getattr(
-                quantization_config, "quant_method", QuantizationMethod.BITS_AND_BYTES
-            )
-
-            if quantization_method_from_args == QuantizationMethod.AWQ:
-                raise ValueError(
-                    "You cannot pass an `AwqConfig` when loading a model as you can only use AWQ models"
-                    " for inference. To quantize transformers models with AWQ algorithm, please refer to our"
-                    " quantization docs: https://huggingface.co/docs/transformers/main_classes/quantization "
-                )
```

Comment:
We can keep this but just check is the model is not already quantized by checking `quantization_method_from_config == QuantizationMethod.AWQ`

Reply:
Makes sense, I added that logic in [0697687](https://github.com/huggingface/transformers/pull/27411/commits/06976877955b57078ffec1acec7aa25d17d1714c) with an additional test in [12aff7c](https://github.com/huggingface/transformers/pull/27411/commits/12aff7c3a52ad0d693e07b03cc98dde30a0c999d)